### DISCUSSION TITLE
Does Nuxt Bridge support Vue 3?

### DISCUSSION THREAD

**Discussion by Dupanoodle** [nuxt/nuxt#18226] [Questions] [answered]
Specifically we want to pull in Vue 3 only packages/modules for Tailwind (HeadlessUI & Heroicons).

A quick test is showing the same failusre as if Vue3 isn't being used however the descrition for Bridge on the landing page says: _"We moved to Vue 3 and re-wrote Nuxt after 4 years of development to make it a strong foundation for the future."_ https://v3.nuxtjs.org/

Could someone confirm if Vue3 is available and that I should be able to use a Vue3 package by including it in the "buildModules' config? Thanks.

**Comment by adarshmadrecha** [comment, marked as answer]
Nuxt Bridge uses Vue 2
Nuxt 3 uses Vue 3
